src/chrome/content/rules/KOOORA.com.xml

```diff
@@ -12,7 +12,8 @@
 	<target host="www.kooora.com" />
 	<target host="m.kooora.com" />
 	<target host="o.kooora.com" />
-
+	<target host="img.kooora.com" />
```

Comment:
Targets should be [sorted alphabetically](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering) (except `www` which comes right after `^`).